lerobot/common/policies/diffusion/modeling_diffusion.py

```diff
@@ -749,3 +768,264 @@ def forward(self, x: Tensor, cond: Tensor) -> Tensor:
         out = self.conv2(out)
         out = out + self.residual_conv(x)
         return out
+
+
+class TransformerForDiffusion(nn.Module):
+    def __init__(self, config: DiffusionConfig, cond_dim: int):
+        super().__init__()
+        self.config = config
+
+        # compute number of tokens for main trunk and condition encoder
+        if config.n_obs_steps is None:
```

Comment:
config.n_obs_steps is None does not seem to be allowed according to the type hinting and documentation. So perhaps it doesn't make sense to handle it here, right?